geode-unsafe/src/main/java/org/apache/geode/unsafe/internal/sun/misc/Signal.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.geode.unsafe.internal.sun.misc;
+
+import java.util.IdentityHashMap;
+
+public class Signal {
```

Comment:
This looks like a brand new class and functionality. Probably needs a unit test? Where is this used?

Reply:
I went back and forth on that. It's really just a 1:1 wrapper around the internal API Signal classes to avoid the compiler warnings on our other modules. I suppose I could whip one up anyway. Thanks!